specs/phase0/p2p-interface.md

```diff
@@ -550,20 +550,18 @@ Clients MUST treat as valid any byte sequences.
 ### Encoding strategies
 
 The token of the negotiated protocol ID specifies the type of encoding to be used for the req/resp interaction.
-Two values are possible at this time:
+Only one value is possible at this time:
 
--  `ssz`: the contents are [SSZ-encoded](../../ssz/simple-serialize.md).
-  This encoding type MUST be supported by all clients.
+-  `ssz_snappy`: The contents are first [SSZ-encoded](../../ssz/simple-serialize.md)
+  and then compressed with [Snappy](https://github.com/google/snappy) frames compression.
   For objects containing a single field, only the field is SSZ-encoded not a container with a single field.
-  For example, the `BeaconBlocksByRoot` request is an SSZ-encoded list of `Root`'s.
```

Comment:
Why remove this? It clarifies something that is still important to ssz-snappy.

Reply:
yeah, you're right. I misread it and thought it was a highly obvious statement (it's not)